src/Forms/HTMLEditor/HTMLEditorSanitiser.php

```diff
@@ -331,6 +344,20 @@ public function sanitise(HTMLValue $html)
                     $el->setAttribute($attr, $forced);
                 }
             }
+
+            // add rel="noopener, noreferrer" to link elements with a target attribute
+            if ($linkNoopenerValue && $el->tagName === 'a') {
+                // user has checked the checkbox 'open link in new window'
+                if ($el->getAttribute('target') && $el->getAttribute('rel') !== $linkNoopenerValue) {
+                    $el->setAttribute('rel', $linkNoopenerValue);
+                } else {
+                    // user previously checked 'open link in new window' and noopener was added,
+                    // now user has unchecked the checkbox so we can remove noopener
+                    if ($el->getAttribute('rel') === $linkNoopenerValue && !$el->getAttribute('target')) {
```

Comment:
This `if` can be combined with the `else` above.

Reply:
done